webthing/server.py

```diff
@@ -364,7 +364,7 @@ def put(self, thing_id='0', property_name=None):
             try:
                 thing.set_property(property_name, args[property_name])
             except PropertyError:
-                self.set_status(403)
+                self.set_status(400)
```

Comment:
It would be nice to expose the PropertyError's message

Reply:
See comment in mozilla-iot/webthing-node#42